tests/processes/ProcessingTest.py

```diff
@@ -546,7 +556,7 @@ def test_yield_ignore_ranges(self):
         self.assertEqual(source_range.start.line, 1)
         self.assertEqual(source_range.start.column, 1)
         self.assertEqual(source_range.end.line, 1)
-        self.assertEqual(source_range.end.column, 14)
+        self.assertEqual(source_range.end.column, 15)
```

Comment:
I suspect that this shouldnt be done.  The `# This case was a bug.` above makes me worried.

It seems like the change is due to an added `\n` in the file.

Reply:
It was https://github.com/coala-analyzer/coala/issues/2563

Reply:
It doesnt seem that missing EOL@EOF was relevant to that bug.

And a missing trailing `\n` is not a thing in universal newlines mode so it shouldnt be a problem ... so ... accepting this change